components/Button/Button.tsx

```diff
@@ -39,7 +39,7 @@ export const SwishButton = ({
                 <div className={styles.swishBackground} />
             </div>
             <div className={styles.text}>{children}</div>
-            <ChevronIcon className={styles.chevron} />
+            <ChevronIcon className={styles.chevron} width={20} height={30} />
```

Comment:
**Suggestion:** Since you've added viewbox to the SVG, it might be better to pass in a "size" attribute with the same value to apply to width and height.

Reply:
Nice, will do